tests/01-api.js

```diff
@@ -120,4 +121,55 @@ describe('bedrock-messages-client API requests', function() {
       });
     });
   });
+
+  describe('start API', function() {
+    before(function() {
+      helpers.removeCollections(function() {
+      });
+    });
```

Comment:
This looks like it's supposed to be async, but isn't because the callback isn't used properly. Should this be:

```
before(function(done) {
  helpers.removeCollections(done);
});
```
?

The same applies to `after`. Look up the docs on async `before` and `after` for mocha and make sure we're doing this part properly.